trade-finance-manager-ui/server/controllers/case/amendments/amendmentBankReviewDate.controller.js

```diff
@@ -22,7 +23,7 @@ const getAmendmentBankReviewDate = async (req, res) => {
     return res.redirect('/not-found');
   }
 
-  if (!changeCoverEndDate || !isTfmFacilityEndDateFeatureFlagEnabled()) {
+  if (!changeCoverEndDate || !isFacilityEndDateEnabledForTfmFacility(facility)) {
```

Comment:
can this be a const for isFacilityEndDateEnabledForTfmFacility

Reply:
see above